src/main/webapp/dataset.xhtml

```diff
@@ -37,10 +37,10 @@
                                                                    or (DatasetPage.workingVersion.deaccessioned and DatasetPage.canUpdateDataset()))}"/>
             <!-- TO-DO FIX RSYNC LOGIC !fileMetadata.dataFile.filePackage or
                                                     fileMetadata.dataFile.filePackage and systemConfig.HTTPDownload -->
-            <ui:param name="showAccessFileButtonGroup" value="#{fileDownloadHelper.canDownloadFile(fileMetadata)
-                                                                or (!fileDownloadHelper.canDownloadFile(fileMetadata) and DatasetPage.dataset.fileAccessRequest)
-                                                                and (!DatasetPage.workingVersion.deaccessioned
-                                                                or (DatasetPage.workingVersion.deaccessioned and permissionsWrapper.canIssueUpdateDatasetCommand(FilePage.fileMetadata.datasetVersion.dataset)))}"/>
+            <!-- NOTE removed fileAccessRequest logic in order to always display the file access btn -->
```

Comment:
Often we put comments like this within a ui:remove so it doesn't show in browser under "view source"

Reply:
That's true. I didn't add this one. Future cleanup?

Reply:
If you feel like adding now, go for it; if not it's not critical (the comment doesn't "leak" anything it shouldn't).

Reply:
Leaving this as future cleanup is fine for me.